sonar-scanner-engine/src/main/java/org/sonar/scanner/analysis/DefaultAnalysisMode.java

```diff
@@ -36,17 +35,12 @@
   private static final Logger LOG = LoggerFactory.getLogger(DefaultAnalysisMode.class);
   private static final String KEY_SCAN_ALL = "sonar.scanAllFiles";
 
-  private boolean mediumTestMode;
   private boolean scanAllFiles;
 
   public DefaultAnalysisMode(GlobalProperties globalProps, AnalysisProperties props) {
     init(globalProps.properties(), props.properties());
   }
 
-  public boolean isMediumTest() {
-    return mediumTestMode;
-  }
-
   public boolean scanAllFiles() {
     return scanAllFiles;
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Add the "@Override" annotation above this method signature [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3AS1161)